pymbar/mbar.py

```diff
@@ -114,12 +114,12 @@ def __init__(self, u_kn, N_k, maximum_iterations=10000, relative_tolerance=1.0e-
         initial_f_k : np.ndarray, float, shape=(K), optional
             Set to the initial dimensionless free energies to use as a
             guess (default None, which sets all f_k = 0)
-        method : list(dict), optional, default=None
+        solver_protocol : list(dict) or None, optional, default=None
             List of dictionaries to define a sequence of solver algorithms
             and options used to estimate the dimensionless free energies.
             See `pymbar.mbar_solvers.solve_mbar()` for details.  If None,
             use the developers best guess at an appropriate algorithm.
```

Comment:
Do you want to be more specific about "developers' best guess? Is it just automatically adaptive?

Reply:
I'll add some more info about what "default" is